docs/backend/schema.ddl

```diff
@@ -26,7 +26,8 @@ Table TypeOfData {
   // Types of data people can view.  This table should be pre-populated with at least the following entries:
   //  1. Trees
   //  2. Sequences
-  //  3. Metadata
+  //  3. Other metadata
```

Comment:
seems like this is wrong based on the enum values

Reply:
Clarified.